### ISSUE TITLE
Questions about reproducing the results

### ISSUE THREAD

**Issue by MichaelCaohn** [google-research-datasets/C4_200M-synthetic-dataset-for-grammatical-error-correction#4]
Hi, Thanks for the great work, and thanks for making this dataset public.

However, I could not find detailed configuration for the pretraining process and training process like the learning rate, optimizer type, learning rate scheduler, warmup steps, drop out rates, etc.

Would you mind sharing these detailed configurations? Such that we can reproduce the result easily and it would be quite expensive for us to search for these configurations during the pretraining stage.

Looking forward to your reply! 

Thank you in advance!

**Comment by mrqorib** [comment]
Hi authors, could you please share more details about the pre-training? It would be helpful to share the pre-training hyper-parameters so that others can reproduce the result. Thank you!

**Comment by fstahlberg** [comment]
Hi,

the *corruption* model that generated the dataset is a seq2edits [1] model trained on the data described in Sec. 4.1 of [2].

For the *correction* (GEC) results in our paper [2] we used vanilla seq2seq transformer_big in Tensor2Tensor (see beginning of Sec. 4.1) with default hyper-parameters. You can use the t2t hyper-parameters as a reference, but for new experiments I wouldn't necessarily recommend using Tensor2Tensor because it is in maintenance mode. My guess would be that C4200M is so large that the default hyper-parameters of any of the seq2seq libraries out there should work fine.

Best, Felix

[1] https://aclanthology.org/2020.emnlp-main.418.pdf
[2] https://aclanthology.org/2021.bea-1.4.pdf
